src/hooks/useActions.ts

```diff
@@ -754,13 +761,24 @@ async function sendIntent(
   }
 }
 
-function imposeGasLimit(intent, gasLimit) {
+function imposeGasLimit(intent: IntentType, gasLimit: number) {
```

Comment:
here the same, i would't define a type for an intent, it can be anything

Reply:
Sure. I will fix it now